.github/workflows/main.yml

```diff
@@ -1,23 +1,22 @@
+# For this workflow to work, you must have a SLACK_WEBHOOK secret defined in either the organization or the repository
```

Comment:
I have set a `SLACK_WEBHOOK` value for the entire organization, so no need to worry about this for now.